_data/simple-icons.json

```diff
@@ -5209,6 +5209,11 @@
             "hex": "FD3A5C",
             "source": "https://www.hotjar.com/"
         },
+        {
+            "title": "Hotwire",
+            "hex": "FFE801",
+            "source": "https://hotwired.dev"
```

Comment:
Update from `https://github.com/hotwired` to `https://hotwired.dev` due to failed tests.